pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -317,6 +317,31 @@ func (c *WorkloadMetaCollector) handleKubePod(ev workloadmeta.Event) []*TagInfo
 func (c *WorkloadMetaCollector) handleECSTask(ev workloadmeta.Event) []*TagInfo {
 	task := ev.Entity.(*workloadmeta.ECSTask)
 
+	taskTags := utils.NewTagList()
+
+	// as of Agent 7.33, tasks have a name internally, but before that
+	// task_name already was task.Family, so we keep it for backwards
+	// compatibility
+	taskTags.AddLow("task_name", task.Family)
+	taskTags.AddLow("task_family", task.Family)
+	taskTags.AddLow("task_version", task.Version)
+	taskTags.AddOrchestrator("task_arn", task.ID)
```

Comment:
This adds `task_arn` as an orchestrator tag to `OrchestratorScopeEntityID`. This matches the current behavior as of 7.35 (at least as much as the tagger is concerned), but I'm not sure if it should remain so based on the investigation of these tags being missing. Should I add them separately as low cardinality to `OrchestratorScopeEntityID`, or keeping it as orchestrator cardinality is fine?

Reply:
as documented [here](https://docs.datadoghq.com/agent/amazon_ecs/tags/) 

it is the right thing to add `task_name` `task_family` `task_version` to cardinality `low`